modules/ballerina-tools/conf/logging.properties

```diff
@@ -1,56 +0,0 @@
-############################################################
```

Comment:
Any reason to remove this this file is used by the ballerina script ?

Reply:
This file is not used. This is just a sample copy of a logging.properties copied, as at the bottom you can see 
```
com.xyz.foo.level = SEVERE
```
and log file name is `carbon`.

The core logging.properties file can be found here https://github.com/ballerinalang/ballerina/blob/master/modules/ballerina-logging/src/main/resources/logging.properties .

I actually didnt remove this file, I renamed it to compose-logging.properties and added the new content. This file will be copied to bre/conf folder. The core also copies logging.properties file to bre/conf.